api/.pipeline/config.js

```diff
@@ -83,6 +83,7 @@ const phases = {
     appHost: (isStaticDeployment && staticUrls.dev) || `${appName}-${changeId}-af2668-dev.apps.silver.devops.gov.bc.ca`,
     backboneApiHost: 'https://api-dev-biohub-platform.apps.silver.devops.gov.bc.ca',
     backboneIntakePath: '/api/dwc/submission/intake',
```

Comment:
Would you mind also adjusting the submission intake path while you're at it. I believe it is now `/queue` rather than `/intake`